.travis.yml

```diff
@@ -7,7 +7,7 @@ php:
   - hhvm
 
 env:
-  - SYMFONY_VERSION="~2.3||~3.0"
+  - SYMFONY_VERSION="2.7.*,>2.7.6"
```

Comment:
@lsmith77 i guess it would not be elegant to merge like this, right? what would be the correct thing to do? i don't want to put a `conflicts` into composer.json just because of some deprecations issue. we should probably not even define a symfony version in this case, to have some installs where we actually use whats in composer.json...